lms/envs/aws.py

```diff
@@ -669,6 +669,7 @@
     FIELD_OVERRIDE_PROVIDERS += (
         'ccx.overrides.CustomCoursesForEdxOverrideProvider',
     )
+CCX_MAX_STUDENTS_ALLOWED = ENV_TOKENS.get('CCX_MAX_STUDENTS_ALLOWED', CCX_MAX_STUDENTS_ALLOWED)
```

Comment:
I think whether or not to make this configurable needs to be edX's decision. I'm fine as long as it isn't more than 200. 

@sarina let us know what you think or whose opinions we need to bring in.